developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md

```diff
@@ -0,0 +1,161 @@
+---
+title: text2vec-jinaai
+sidebar_position: 13
```

Comment:
Let's make this 12 just so it doesn't overlap w/ `text2vec-openai`

Reply:
Updated